Tests/scripts/pkg_dev_container_setup.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+# Setup a container for dev testing. 
+#
+# In alpine will install necessary python dev tools and then use pip to install
+
+cp /dev/stdin /pkg-dev-test-requirements.txt
```

Comment:
I think we should add a comment that states from where are we getting the stdin, would help future maintenance of this code

Reply:
added in new commit